Procfile

```
@@ -1,3 +1,3 @@
 release: python manage.py migrate --no-input
 web: gunicorn --bind :$PORT --workers 4 --worker-class saleor.asgi.gunicorn_worker.UvicornWorker saleor.asgi:application
-celeryworker: celery -A saleor --app=saleor.celeryconf:app worker --loglevel=info -E
+celeryworker: celery -A saleor --app=saleor.celeryconf:app worker --loglevel=info -E --concurrency 2
```

[Bigger instance types](https://devcenter.heroku.com/articles/dyno-types) could set a higher concurrency, thus we could make it customizable:
```suggestion
celeryworker: celery -A saleor --app=saleor.celeryconf:app worker --loglevel=info -E --concurrency ${CELERY_CONCURRENCY:-2}
```